### PR TITLE
Add a skip_zeros keyarg to analyze_malloc_fils.

### PR DESCRIPTION
**Pull request by Azzaare**
Default behavior unchanged.

This would solve https://github.com/JuliaConstraints/PerfChecker.jl/issues/38 and provide a similar option for those in need.

A test entry has been added (and passed).

Note that one of the tests (not introduced by this PR) fails on my local machine with Julia 1.10)

Please tell me if such a PR cannot be accepted. And do not hesitate to tag a new version otherwise!

EDIT: I am solving the error for Julia 1.x, x > 9, step by step. Who got the idea to make the Base parser evolve 🤣 

EDIT2: done. This ugly fix tests the error message from process_file against the correct msg in 1.x when x > 9, for both windows and Unix …

### CODE REVIEW

File: src/memalloc.jl [hunk 1]
```diff
@@ -23,6 +24,7 @@ function analyze_malloc_files(files)
                 if !isempty(tln) && isdigit(tln[1])
                     s = split(tln)
                     b = parse(Int, s[1])
+					skip_zeros && b == 0 && continue
```
Comment (IanButterworth) [inline src/memalloc.jl]: Nit but can you use spaces for indent, to match repo style

Comment (Azzaare) [reply src/memalloc.jl]: Oh yeah, I forgot to set indent to spaces at my home computer where I made the changes.

I will do it tomorrow!

Thanks for the review.

Comment (Azzaare) [reply src/memalloc.jl]: Done!

**Comment by Azzaare** [comment]
It should be ready to be merged! If #72 is rebased over main after that, it should pass the tests.

Note that the tests are likely to be broken again with changes in the parser in the future. We could let a note about it somewhere in the README.

**Review by fingolfin — APPROVED** [review]

**Comment by Azzaare** [comment]
Thanks again, both of you @IanButterworth @fingolfin 

Would one of you mind tagging a new version?

**Pull request merged**